cachelib/navy/common/Device.cpp

```diff
@@ -234,6 +242,14 @@ class AsyncIoContext : public IoContext {
  private:
   void handleCompletion(folly::Range<folly::AsyncBaseOp**>& completed);
 
+  std::unique_ptr<folly::AsyncBaseOp>
+   prepAsyncIo(int fd, uint8_t opType, void* data,
+          uint32_t size, uint64_t offset, void* userdata, bool useIoUring);
+
+  std::unique_ptr<folly::AsyncBaseOp>
+    prepUringCmdAsyncIo(int fd, uint8_t opType, void* data,
+      uint32_t size, uint64_t offset, void* userdata, int handle);
```

Comment:
nit. just `prepNvmeIo` or so. Also, can we make this being called inside `prepAsyncIo`?